giraffe/src/components/LineLayer.tsx

```diff
@@ -43,6 +43,10 @@ export const LineLayer: FunctionComponent<Props> = props => {
     shadeAboveY: height,
   }
 
+  if (config.colorMapping) {
```

Comment:
```suggestion
  if (config.colorMapping && config.colorMappingCallback) {
```

let's not infer that the `colorMappingCallback` was passed in if the `colorMapping` is, let's be explicit that they're both there

Reply:
Very true! I'll fix that.